peps/pep-0752.rst

```diff
@@ -93,6 +97,16 @@ similar characters but that is insufficient for these use cases.
 
 __ https://github.com/pypi/warehouse/blob/8615326918a180eb2652753743eac8e74f96a90b/warehouse/migrations/versions/d18d443f89f0_ultranormalize_name_function.py#L29-L42
 
+Another problem that namespacing would solve is the issue of choosing new names
+for packages following the agreed patterns of naming. Often (this is the case
+for ``Apache Airflow`` for example), there are public discussions that precede
+the decision to create a new package. The decision is based on the agreed
+name and follow the pattern of the existing packages. If more package names are
+considered during the discussion, all the names have to be reserved via ``PyPI``
+interface before the discussion is public, otherwise the names can be taken by
+other users. This is a problem that actually happened in the past as explained
+in the associated discussion.
```

Comment:
Can we link to the associated discussion?
```suggestion
other users. This has happened in the past as explained
in the associated discussion.
```

Reply:
I added the discussion link